sockutils.c

```diff
@@ -65,8 +62,12 @@ int sockcount = 0;					/* Variable that allows calling the WSAStartup() only one
 
 /* Some minor differences between UNIX and Win32 */
 #ifdef WIN32
-#define SHUT_WR SD_SEND			/* The control code for shutdown() is different in Win32 */
-#define snprintf _snprintf		/* The snprintf is called _snprintf() in Win32 */
+#define SHUT_WR SD_SEND				/* The control code for shutdown() is different in Win32 */
+#define snprintf(x, y, z, ...) \
```

Comment:
This shouldn't be necessary as of 905467b72b83897216b172f440f38d14ba00953b - everything should be using `pcap_snprintf()`, which is guaranteed to always null-terminate the string.